index.js

```diff
@@ -81,7 +83,7 @@ async function runLinterFromPRData (pullRequests, context, headSha) {
     const banditReport = generateBanditReport(banditResults, cache.getBranchPath(repoID, PR.id, 'head', 'bandit'))
 
     const gosecResults = await runGosec(cache.getBranchPath(repoID, PR.id, 'head', 'gosec'), inputFiles)
-    const gosecReport = generateGosecReport(gosecResults)
+    const gosecReport = generateGosecReport(gosecResults, path.join(__dirname, cache.getBranchPath(repoID, PR.id, 'head', 'gosec')))
```

Comment:
Prefer using `path.resolve` to convert relative paths to absolute ones rather than `__dirname`.

Reply:
There are a few more instances of this in the code that should be changed too.